project/driver_controls/inc/input_event.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+// Define common typedefs for the various FSMs to use.
+
+#include "event_queue.h"
+
+// Typedef for the FSM arbitration functions
+typedef bool (*InputEventCheck)(const Event *e);
+
+// ID definitions for the driver input events.
+typedef enum {
+  INPUT_EVENT_NONE = 0,                     // Pedal         00
```

Comment:
What function do these comments here serve? If the additional meta-information they provide is important, I'd prefer having them captured in code as opposed to comments.

It seems as if you're using these comments to distinguish which input each event is associated with. You could maybe prefix the input source with the event, like you've done with turn signals?